js/dashboard.js

```diff
@@ -906,7 +906,10 @@ var Dashboard = {
    getCardsAjax: function(specific_one) {
       specific_one = specific_one || "";
 
-      var promises = [];
+      var filters = Dashboard.getFiltersFromDB();
```

Comment:
getFiltersFromDB only exists in master and replaces getFiltersFromStorage. I believe that you developed your pr on top of master branch. If I'm right, this PR will break dashboards; you need to call getFiltersFromStorage instead of getFiltersFromDB. See deleted line 924 below.

Reply:
It was. I added a commit to make it work with 9.5. Since the main issue is the synchronous AJAX request in getFiltersFromDB, this will need changed when applied to the master branch. The other difference is using `compile` vs `compileString` in the SCSS class. The first is deprecated in the version used on master but will still work. I switched it back to `compile` to work on both.

Reply:
question about this, as some enhancements have been done in master branch for the dashboard.
Is it mandatory for this pr to target 9.5/bugfixes ?

At least, can we retarget master and work on a backport if needed ?
This steps will ease the integration of the current and other PR. And it will also ease the work of cherry picking the pr in master for mainteners.

Another possibility could be to let this pr ending in 9.5 and you work on the adaptation of master branch in a separate pr.
Anyway, you see the issue i think, the current work targetting the stable branch overlap with existing evolutions.